daemons/dss-sync-sfn/app.py

```diff
@@ -76,6 +77,9 @@ def launch_from_forwarded_event(event, context):
             source_replica = Replica.gcp
             source_key = message["name"]
             bucket = source_replica.bucket
+            if source_key.startswith(BLOB_PREFIX) and not BLOB_KEY_REGEX.match(source_key):
+                logger.info("Key %s does not match blob key format, skipping sync", source_key)
```

Comment:
What's more appropriate: `logger.info` or `logger.warning`?

Reply:
i would think that warnings would be reserved for something that leaves the application in an odd state.
If the standard approach to dealing with a .parts file is ignoring it, i would think that logger.info would be appropriate.